### PR TITLE
#5044 checkbox neo

### LINKED ISSUE #5044: Check box styling
image
should be: 
image

### PR DESCRIPTION
**Pull request by prachi00**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Closes #5044
- [ ] Requires deployment <>

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: 
[Payout](https://beta.kodadot.xyz/transfer/?target=EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.
Screen Shot 2023-02-21 at 7 02 22 PM

### CODE REVIEW

**Comment by prachi00** [comment]
Known issue:
value not getting saved on reload, I tried using nativeValue from oruga but it doesnt work in vue composition, does anyone know any way to do this https://oruga.io/components/checkbox.html

File: libs/ui/src/components/NeoCheckbox/NeoCheckbox.story.vue [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<template>
+  <Story title="NeoButton" :layout="{ type: 'grid', width: '200px' }">
+    <Variant title="small neo button">
+      <NeoCheckbox> test </NeoCheckbox>
```
Comment (roiLeo) [inline libs/ui/src/components/NeoCheckbox/NeoCheckbox.story.vue:4]:
```suggestion
      <NeoCheckbox> {{ label }} </NeoCheckbox>
```

**Review by roiLeo — APPROVED** [review]
✅ code lgtm

**Review by roiLeo — COMMENTED** [review]
Screenshot 2023-02-22 at 09-06-43 Low minting fees and carbonless NFTs KodaDot - NFT Market Explorer

checkmark should be smaller

> Known issue: value not getting saved on reload, I tried using nativeValue from oruga but it doesnt work in vue composition, does anyone know any way to do this https://oruga.io/components/checkbox.html

Isn't it working like `NeoDropdown` component?

**Comment by prachi00** [comment]
not sure how to get smaller checkbox, its a background image in oruga @roiLeo

**Comment by roiLeo** [comment]
> not sure how to get smaller checkbox, its a background image in oruga @roiLeo

```css
background-size: 10px;
background-repeat: no-repeat;
background-position: center;
```

**Review by roiLeo — APPROVED** [review]

**Comment by prachi00** [comment]
can we merge this one?

**Comment by vikiival** [comment]
Pay 40 USD

**Comment by vikiival** [comment]
Pay 40 USD

**Review by vikiival — APPROVED** [review]

**Comment by vikiival** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 38.98 USD/KSM ~ 1.026 $KSM
🧗 [EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU](https://kusama.subscan.io/account/EzGc4s9PgCPx1YnF3fqzhLzVHpHMTL4LWPScwpDrR8JKgSU)
🔗 [0xd3f69165843181633df987ac8c42aa19b9e2be0668197874dc2fe9eeb059ac0c](https://kusama.subscan.io/extrinsic/0xd3f69165843181633df987ac8c42aa19b9e2be0668197874dc2fe9eeb059ac0c)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Comment by prachi00** [comment]
@vikiival @yangwao seems like the payment failed

**Comment by prachi00** [comment]
@vikiival can we retry payment on this?

**Comment by yangwao** [comment]
pay 40 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $40 @ 5 USD/DOT ~ 8 $DOT
🧗 [13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1](https://polkadot.subscan.io/account/13Qx65nLd6SwdtjrRyuoEtp9CKXhF651xdHBPaXcvhwKm4N1)
🔗 [0x9b577a4835227ef8e1744c77fa100253c1b523ea79c63c0490d019bf790d262e](https://polkadot.subscan.io/extrinsic/0x9b577a4835227ef8e1744c77fa100253c1b523ea79c63c0490d019bf790d262e)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**